plugins/GSdx/GSTextureCache.h

```diff
@@ -104,11 +105,12 @@ class GSTextureCache
 		void Update();
 	};
 
+	using FastSourcePtrList = FastList<Source*, 4>;
```

Comment:
Maybe a type definition might be more suitable? (I'm unsure, I've never really seen anyone exploiting the ``using`` keyword this way)

```cpp
typedef FastList<Source*, 4> FastSourcePtrList;
```